content/home.js

```diff
@@ -11,8 +11,69 @@ module.exports = {
     title: `Gold Demands <br/> 2022`,
     subtitle: 'Subtitle',
   },
+  introArticle: {
+    title: 'Key insight 3',
+    text:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Bibendum est ultricies integer quis. Iaculis urna id volutpat lacus laoreet. Mauris vitae ultricies leo integer malesuada. Ac odio tempor orci dapibus ultrices in. Egestas diam in arcu cursus euismod.\n' +
+      'Dictum fusce ut placerat orci nulla. Tincidunt ornare massa eget egestas purus viverra accumsan in nisl. Tempor id eu nisl nunc mi ipsum faucibus. Fusce id velit ut tortor pretium. Massa ultricies mi quis hendrerit dolor magna eget.',
+  },
+  articleOne: {
```

Comment:
article might not be the correct name for this, as the whole page is basically an article